ansible/roles/vm_set/library/vm_topology.py

```diff
@@ -941,10 +940,13 @@ def main():
         supports_check_mode=False)
 
     cmd = module.params['cmd']
-    vm_names = module.params['vm_names']
     fp_mtu = module.params['fp_mtu']
     max_fp_num = module.params['max_fp_num']
     duts_mgmt_port = []
+    if 'vm_names' not in module.params:
+        vm_names = []
+    else:
+        vm_names = module.params['vm_names']
```

Comment:
As above.

Reply:
Hi @yxieca 

I have updated vm_topology.py as per the review comments. 
Please share your feedback on the latest changes.